packages/create-svelte/templates/default/src/routes/todos/index.svelte

```diff
@@ -105,12 +105,13 @@
 				action="/todos/{todo.uid}.json?_method=delete"
 				method="post"
 				use:enhance={{
+					pending: () => todo.pending_delete = true,
 					result: () => {
 						todos = todos.filter((t) => t.uid !== todo.uid);
 					}
 				}}
 			>
-				<button class="delete" aria-label="Delete todo" />
+				<button class="delete" aria-label="Delete todo" disabled={!!todo.pending_delete} />
```

Comment:
```suggestion
				<button class="delete" aria-label="Delete todo" disabled={todo.pending_delete} />
```

You'll need to update the types above as well for TS users